<launch>
  <!--  Turtlebot_kinect_lidar Gazebo simulation -->
  <include file="$(find turtlebot_custom)/launch/Borealis/borealis_turtlebot_gazebo.launch"/>

  <!-- Simulation Robot Pose Publisher -->
  <node pkg="follow_me" type="pose_publisher.py" name="gazebo_pose_publisher" output="screen"/>

  <!--  Custom Rviz -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find follow_me)/launch/simulation/debug.rviz"/>

  <!--  Teleop joystick PS4 -->
  <include file="$(find follow_me)/launch/simulation/ps4_joystick.launch"/>

  <!-- Pose to Path publisher -->
  <include file="$(find pose_to_path)/launch/pose_to_path.launch"/>


  <!-- UAV1 Follower -->
  <!--include file="$(find follow_me)/launch/follow_me.launch">
    <arg name="robot_name" value="uav1" />
    <arg name="robot_pose_topic" value="/UAV1Pose" />
    <arg name="follow_distance" value="1.0" />
  </include-->


  <!-- UAV2 Follower -->
  <!--include file="$(find follow_me)/launch/follow_me.launch">
    <arg name="robot_name" value="uav2" />
    <arg name="robot_pose_topic" value="/UAV2Pose" />
    <arg name="follow_distance" value="2.0" />
  </include-->

  <!-- UAV3 Follower -->
  <!--include file="$(find follow_me)/launch/follow_me.launch">
    <arg name="robot_name" value="uav3" />
    <arg name="robot_pose_topic" value="/UAV3Pose" />
    <arg name="follow_distance" value="3.0" />
  </include-->

  
</launch>
